ports/mimxrt/modules/_boot.py

```diff
@@ -3,13 +3,31 @@
 # Note: the flash requires the programming size to be aligned to 256 bytes.
 
 import os
+import sys
 import mimxrt
+from machine import SDCard
 
 bdev = mimxrt.Flash()
-
 try:
     vfs = os.VfsLfs2(bdev, progsize=256)
 except:
     os.VfsLfs2.mkfs(bdev, progsize=256)
     vfs = os.VfsLfs2(bdev, progsize=256)
-os.mount(vfs, "/")
+finally:
+    os.mount(vfs, "/flash")
+    os.chdir("/flash")
+    sys.path.append("/flash")
+
+# do not mount the SD card if SKIPSD exists.
```

Comment:
Does it make sense to automatically mount the SD card?  It might be better to just leave it up to the user to mount it in their own `boot.py` or `main.py` when/if they want to use it (that's how some stm32 boards work).

Reply:
robert_hh: That's the way Pyboards work. You can have a SKIPSD file in flash to avoid SD Card mounting.

Reply:
I fixed the startup behavior now. If there is no `SDCard` module or `SKIPSD` is set then there will be no mounting. I think it is nice from a users perspective to automatically mount the card so I would keep it for now.